### ISSUE TITLE
REQ: Remove-Service cmdlet

### ISSUE THREAD

**Issue by chucklu** [PowerShell/PowerShell#4717]
Hi,
    Since we have get-service,start-service,and some other functions about windows service.
    Is it possible to add intall-service and uninstall-service?
   
 I am using installutil.exe tool to do the installation and uninstallation.
    sometimes, the uninstallation did not work.
```
installutil /u 
```
   

   then I need to switch to cmd prompt,and use 
```
 sc delete
```

   If you can support install and uninstall service,it will be great.

Environment data
----------------

```powershell
> $PSVersionTable
Name                           Value
----                           -----
PSVersion                      6.0.0-beta
PSEdition                      Core
GitCommitId                    v6.0.0-beta.6
OS                             Microsoft Windows 10.0.10586
Platform                       Win32NT
PSCompatibleVersions           {1.0, 2.0, 3.0, 4.0...}
PSRemotingProtocolVersion      2.3
SerializationVersion           1.1.0.1
WSManStackVersion              3.0
```

**Comment by chucklu** [comment]
By the way, when the installutil.exe /u did not work. It is something like [this](https://stackoverflow.com/questions/13608157/installutil-wont-uninstall-the-specified-service-does-not-exist-as-an-install).

**Comment by chuanjiao10** [comment]
Install-Service named New-Service:
name    New-Service
Creates a new Windows service.

but ps team not give us remove-service.

**Comment by chucklu** [comment]
@chuanjiao10  thanks,new-service works.

**Comment by SteveL-MSFT** [comment]
There isn't a `Remove-Service`.  If needed, it would be an enhancement.

**Comment by chucklu** [comment]
@SteveL-MSFT  Would you add remove-service to an enhancement list?

**Comment by SteveL-MSFT** [comment]
@chucklu I've updated the title of this to reflect the ask, currently no plans to implement this

**Comment by chucklu** [comment]
@SteveL-MSFT  Thanks,I wish you guys can implement it  in the future.

**Comment by SteveL-MSFT** [comment]
@chucklu workaround for now:

```powershell
get-ciminstance win32_service -filter "name='yourservicename'" | remove-ciminstance
```

**Comment by chucklu** [comment]
@SteveL-MSFT  thanks,I also find the command in powershell [doc](https://docs.microsoft.com/en-us/powershell/module/microsoft.powershell.management/new-service?view=powershell-6)
```powershell
PS C:\> sc.exe delete TestService
- or -
PS C:\> (Get-WmiObject win32_service -Filter "name='TestService'").delete()
```
It's not convenient, if you can support _Remove-Service_, it would be great.

**Comment by SteveL-MSFT** [comment]
Get-WmiObject is deprecated and replaced by Get-CimInstance

**Comment by chucklu** [comment]
thanks

**Comment by adityapatwardhan** [comment]
Closed by #4858
